rootfs/etc/nginx/lua/metric_emitter.lua

```diff
@@ -0,0 +1,50 @@
+local socket = ngx.socket.udp
+local cjson = require('cjson')
+local assert = assert
+
+local _M = {
+  queue = {}
+}
+
+local function flush_queue()
+  local s = assert(socket())
+  assert(s:setpeername("127.0.0.1", 8000))
+  for _, v in ipairs(_M.queue) do
+    assert(s:send(v))
+  end
+  _M.queue = {}
+  assert(s:close())
+end
+
+function _M.call()
+  -- Create JSON Metrics Payload  --
+  local jsonPayload = cjson.encode({
+    host                 = ngx.var.host                             or "-",
+    status               = ngx.var.status                           or "-",
+    remoteAddr           = ngx.var.realip_remote_addr               or "-",
+    remoteUser           = ngx.var.remote_user                      or "-",
+    bytesSent            = tonumber(ngx.var.bytes_sent)             or -1,
+    protocol             = ngx.var.server_protocol                  or "-",
+    method               = ngx.var.request_method                   or "-",
+    path                 = ngx.var.uri                              or "-",
+    requestLength        = tonumber(ngx.var.request_length)         or -1,
+    requestDuration      = tonumber(ngx.var.request_time)           or -1,
+    upstreamName         = ngx.var.upstream                         or "-",
+    upstreamIP           = ngx.var.upstream_addr                    or "-",
+    upstreamResponseTime = tonumber(ngx.var.upstream_response_time) or -1,
+    upstreamStatus       = ngx.var.upstream_status                  or "-",
+    namespace            = ngx.var.namespace                        or "-",
+    ingress              = ngx.var.ingress_name                     or "-",
+    service              = ngx.var.service_name                     or "-",
```

Comment:
can you avoid spacing like this? later on when we add another line that's longer we then will have to adjust all the others

Reply:
The stats on the list were given by @aledbf on the kubernetes slack channel. This does not yet cover all VTS metrics.

Reply:
@fmejia97 feel free to add all the fields you think are required to match the VTS module. I just only added the minimum to start the work with the UDP server